src/app/child-dev-project/schools/display-participants-count/display-participants-count.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
+  {{ participantRelationsCount }}
```

Comment:
Do you have insights, how expensive it is to call a getter in a template? So far we have tried to get rid of function call in templates (#1087) as they might end up getting called hundreds of times. So far we mostly just made the private property (`_participantRelationsCount`) public to directly access it in the template.

Reply:
Good point, i remember darkly... To avoid the getter and also to get rid of the ChangeDetection injection, i just use a Signal here. I just read an article some weeks ago, that they got rid of unnecessary calculations within the template when you use Signals.